deployment/postgres-init-db/sql/sched/tables/scheduling_uploaded_file.sql

```diff
@@ -0,0 +1,36 @@
+create table scheduling_uploaded_file (
```

Comment:
This table is maybe not needed for the scheduler server, but this exists to show that providing a custom prefix to `uploaded_file` works around Hasura's duplicate table metadata issue. This table can be removed as part of this PR if desired, but for now it serves as an example of our Hasura workaround.

Reply:
What do you think of pulling this out into a separate branch for now? We can hold it there as an example until we determine if we need to use this approach.

Reply:
That's fair, I'll take it out of this PR.